norns/play.md

```diff
@@ -331,8 +331,26 @@ To clear the script running on norns:
 - while holding **K1**, press **K3** to clear the currently-running script
 - release **K1** and you'll see `NONE` where the previously-running script's name used to be
 
+## SYSTEM
+
+![](/docs/norns/image/play-images/system.png)  
+[*figure 9: the norns SYSTEM menu*](image/play-images/system.png)
+
+The SYSTEM menu contains a number of helpful entries:
+
+- `DEVICES` allows you to manage all your connected devices (MIDI, grid, arc, and HID). See [control + clock](/docs/norns/control-clock/) for more info.
+- `WIFI` manages network connectivity, including hotspot mode, wireless access, and ethernet. See [WiFi / update / files](/docs/norns/wifi-files/) for more info.
+- `MODS` are small chunks of code to create custom modifications to the core workings of the norns system software. See [mods](/docs/norns/community-scripts/#mods) for more info.
+- `SETTINGS` allows you to:
+  - `RESET` your norns back to its workshop settings, erasing your favorites, disabling all mods, and restoring global audio levels to their defaults.
```

Comment:
do you mean _workshop_? (or maybe "default", "factory" or something?)

Reply:
oh! lol i always circumnavigate 'factory' with 'workshop', to try and set the right mental image of our operations, but will change to 'default', thank you!